src/main.cpp

```diff
@@ -119,13 +113,7 @@ QCoreApplication* createApplication(int& argc, char* argv[], unsigned int& actio
     }
 
     if ((action) & (1 | 2)) {
-        // Ah, we're gonna bail out early, if Windows we still need a GUI Application
```

Comment:
Might like to leave a comment to the effect:
```
    // Ah, we're gonna bail out early, just need a command-line application
```
~~because we check whether we have a `QApplication` pointer (and not a `QCoreApplication`) back in the caller to this function~~ oh, no we do not :open_mouth: ...